### PR TITLE
Taxprofiler: Update databases to include db_type

### PR DESCRIPTION
**Pull request by LilyAnderssonLee**

### CODE REVIEW

File: database_full_v1.2.csv [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+diamond,diamond-db3,,both,s3://ngi-igenomes/test-data/taxprofiler/diamond.dmnd
+kaiju,kaiju-db1,,short,s3://ngi-igenomes/test-data/taxprofiler/kaiju.tar.gz
+kaiju,kaiju-db2,-m 50,long,s3://ngi-igenomes/test-data/taxprofiler/kaiju.tar.gz
+kaiju,kaiju-db3,,both,s3://ngi-igenomes/test-data/taxprofiler/kaiju.tar.gz
+kraken2,kraken2-db,,short,s3://ngi-igenomes/test-data/taxprofiler/kraken2.tar.gz
+krakenuniq,krakenuniq-db,,short,s3://ngi-igenomes/test-data/taxprofiler/krakenuniq.tar.gz
+malt,malt-db,,short,s3://ngi-igenomes/test-data/taxprofiler/malt.tar.gz
+metaphlan,metaphlan3-db,--mpa3,short,s3://ngi-igenomes/test-data/taxprofiler/metaphlan3.tar.gz
+motus,motus-db1,,short,s3://ngi-igenomes/test-data/taxprofiler/db_mOTU.tar.gz
+motus,motus-db2,prep_long,long,s3://ngi-igenomes/test-data/taxprofiler/db_mOTU.tar.gz
```
Comment (LilyAnderssonLee) [inline database_full_v1.2.csv:15]: I am not sure if it's correct to include `prep_long` here.

Comment (sofstam) [reply database_full_v1.2.csv:15]: How do you mean?

Comment (LilyAnderssonLee) [reply database_full_v1.2.csv:15]: Not sure if that's the correct way to define in `db_params`

Comment (sofstam) [reply database_full_v1.2.csv:15]: Ok!

**Review by sofstam — APPROVED** [review]

**Pull request merged**